files/teaching-demo-rubric.md

```diff
@@ -3,28 +3,28 @@ This rubric is provided as a guide for Trainers evaluating potential new instruc
+mark in a "Negative" column can be considered justification for asking a trainee to redo their demonstration if the problem is significant. As always,
 Trainers should use their own judgement when applying this rubric in individual cases.  
 
 |Positive Content|Negative Content|
 |------|---------------------|
-Trainee uses the official curricular materials as written or with minor deviations|Trainee deviates significantly from the official curricular materials|
-Trainee places content in context and explains why it is relevant/useful to the learner|Trainee jumps into the content without explaining context|
-Trainee obviously knows the material they are teaching|Trainee makes factual errors in teaching the content and doesn't correct themselves|
+Uses official curriculum with only minor deviations|Deviates significantly or doesn't use official curriculum|
+Places content in context and explains relevance/utility to learners|Jumps into the content without context|
+Teaches content correctly|Makes factual errors in content and doesn't correct|
 
 |Positive Delivery|Negative Delivery|
 |------|---------------------|
-Trainee mirrors the learner's environment (e.g. default terminal setup, simplified shell prompt)|Trainee uses shortcuts or tools not immediately available to or familiar to the learner|
-Trainee uses appropriately sized fonts and windows|Font or windows are too small|
-Trainee explains everything that they type|Trainee trouble-shoots an error or otherwise types commands without explaining them to the learner|
-Trainee uses mistakes/typos as opportunities for learning|Trainee ignores or dismisses mistakes/typos that they make|
--|Trainee displays the lesson materials/notes on their screen while teaching|
--|Trainee has notifications show up on their screen while teaching|
--|Trainee uses significant amounts of dismissive language (e.g. "just do this", "easy", "simple")|
-Trainee speaks and types at an appropriate pace|Trainee speaks or types very quickly|
-Trainee speaks clearly|-
--|Trainee appears significantly distracted|
+Mirrors the learner's environment (e.g. default terminal setup, simplified prompt)|Uses shortcuts or tools that are unfamiliar/unavailable to learners|
+Uses appropriately sized fonts and windows|Font or windows are too small|
+Explains all typing|Types commands or troubleshoots errors without explaining|
+Uses mistakes/typos as opportunities for learning|Ignores or dismisses mistakes/typos that they make|
+Uses notes on paper/tablet|Displays lesson materials/notes on their screen|
+-|Notifications show up on their screen|
+-|Uses significant amounts of dismissive language (e.g. "just", "easy")|
```

Comment:
- Would add something about consulting the notes too often - I've had demos in which the instructor was checking every few seconds.
- Additionally, should there be an item for "seems to know the material well" vs. "seems unfamiliar with the material"? (the latter may have inspired the frequent note checking mentioned above, or that may have been a nervous tic)